src/main.rs

```diff
@@ -9,10 +10,9 @@ fn main() -> Result<()> {
 	);
 
 	let raw = fs::read_to_string("examples/scoring.toml")?;
-	let config: Config = toml::from_str(&raw)?;
-
-	for scored in config.score() {
-		println!("Check passed: {} - {} points", scored.name, scored.points);
-	}
+  let config: Config = toml::from_str(&raw)?;
+  
+  write_to_scoring_report(&String::from("/opt/chiron/scoringreport.html"), config.score());
```

Comment:
no need for the heap-allocated string here, you can just do `"/opt/chiron/scoringreport.html"`

Reply:
plus this'll probably be a constant so put like `const SCORING_REPORT: &str = "file"` in `lib.rs`